job-runner/runner/runner.go

```diff
@@ -145,6 +159,8 @@ TRY_LOOP:
 			break TRY_LOOP
 		}
 
+		tryNo++
```

Comment:
Put this after the `// Wait between retries` block because in the rare case that the job is stopped while waiting, this would cause the wrong job_try to be reported. For example, let's say maxTries = 2 and try 1 fails, then we tryNo++ here so now job_try=2, then we wait, but the job is stopped during that wait, we'll return Tries=2 when only 1 try happened.

Reply:
oof good catch. thank you.